mart/configs/optimizer/sgd.yaml

```diff
@@ -1,9 +1,3 @@
-_target_: mart.optim.OptimizerFactory
-optimizer:
-  _target_: hydra.utils.get_method
-  path: torch.optim.SGD
+_target_: torch.optim.SGD
+_partial_: true
 lr: ???
-momentum: 0
-weight_decay: 0
-bias_decay: 0
-norm_decay: 0
```

Comment:
I wouldn't change this given that it's often useful to set decay values in configuration.

Reply:
OK. I made the simplified config template a comment.